src/LaserTag.jl

```diff
@@ -143,6 +137,17 @@ function POMDPs.reward(p::LaserTagPOMDP, s::LTState, a::Int, sp::LTState)
     end
 end
 
+
+# add a function to transform a LTState to a vector 
+function POMDPs.convert_s(T::Type{<:AbstractArray}, s::LTState, p::LaserTagPOMDP)
+    return convert(T, vcat(s.robot, s.opponent, s.terminal))
+end
+
+# transform a vector to a LTState
+function POMDPs.convert_s(T::Type{LTState}, v::AbstractArray{Float64}, p::LaserTagPOMDP)
+    return LTState([v[1], v[2]], [v[3], v[4]], v[5])
```

Comment:
```suggestion
    return LTState(Coord(v[1], v[2]), Coord(v[3], v[4]), v[5])
```

Reply:
Hello @zsunberg! Sorry, I totally forgot this change after Xmas... I updated and pushed it! Hope what did is correct.
Pls let me know if everything is ok.